app/lib/mappers/mapServices.js

```diff
@@ -0,0 +1,59 @@
+const utils = require('../utils');
+
+const EPS_ENABLED = 'epsIsEnabled';
+const MORE_INFORMATION = 'servicesMoreInformationText';
+const TEXT = 's:text';
+const TYPE = 's:type';
+const SUMMARY = 's:servicesummary';
+const AVAILABILITY_TIMES = 's:serviceBranchAvailabilityTimes';
+const INTRODUCTION = 's:serviceIntroduction';
+const GP_REFERRAL_REQUIRED = 's:serviceGpReferralRequired';
+const DELIVERER = 's:serviceDeliverer';
+const NAME = 's:name';
+
+function getText(member) {
+  return member && member[TEXT];
+}
+
+function getUnderscore(member) {
+  return member && member._;
+}
+
+function epsEnabled(eps) {
+  return eps && eps._ && eps._.toLowerCase() === 'true';
```

Comment:
This `eps._.toLowerCase() === 'true'` could be `new Boolean(eps._).toString() === 'true'` which should still work regardless of the casing of `eps._`